Documentation/operations/metrics.rst

```diff
@@ -286,146 +285,145 @@ report the eBPF memory usage directly managed by Cilium.
-``ip_addresses``                         ``family``                                   Number of allocated IP addresses
-======================================== ============================================ ========================================================
+======================================== ============================================ ========== ========================================================
+Name                                     Labels                                       Default    Description
+======================================== ============================================ ========== ========================================================
+``ipam_events_total``                                                                 Enabled    Number of IPAM events received labeled by action and datapath family type
+``ip_addresses``                         ``family``                                   Enabled    Number of allocated IP addresses
+======================================== ============================================ ========== ========================================================
 
 KVstore
 ~~~~~~~
 
-======================================== ============================================ ========================================================
-Name                                     Labels                                       Description
-======================================== ============================================ ========================================================
-``kvstore_operations_duration_seconds``  ``action``, ``kind``, ``outcome``, ``scope`` Duration of kvstore operation
-``kvstore_events_queue_seconds``         ``action``, ``scope``                        Duration of seconds of time received event was blocked before it could be queued
-``kvstore_quorum_errors_total``          ``error``                                    Number of quorum errors
-======================================== ============================================ ========================================================
+======================================== ============================================ ========== ========================================================
+Name                                     Labels                                       Default    Description
+======================================== ============================================ ========== ========================================================
+``kvstore_operations_duration_seconds``  ``action``, ``kind``, ``outcome``, ``scope`` Enabled    Duration of kvstore operation
+``kvstore_events_queue_seconds``         ``action``, ``scope``                        Enabled    Duration of seconds of time received event was blocked before it could be queued
+``kvstore_quorum_errors_total``          ``error``                                    Enabled    Number of quorum errors
```

Comment:
:+1: actually yeah these ones are likely also only enabled by default if you configure a kvstore instead of k8s as the primary method of state distribution. LGTM.